ERROR tests/test_code_conventions.py::test_datasheet_yamllint

There appeared pytest error:
```
============================= test session starts ==============================
platform linux -- Python 3.10.2, pytest-7.0.1, pluggy-1.0.0 -- /usr/bin/python3
cachedir: .pytest_cache
rootdir: /usr/src/RPM/BUILD/liboqs-0.7.1
plugins: xdist-2.5.0, forked-1.4.0
gw0 I / gw1 I / gw2 I / gw3 I / gw4 I / gw5 I / gw6 I / gw7 I
...
==================================== ERRORS ====================================
__________________ ERROR at setup of test_datasheet_yamllint ___________________
[gw3] linux -- Python 3.10.2 /usr/bin/python3

cls = <class '_pytest.runner.CallInfo'>
func = <function call_runtest_hook.<locals>.<lambda> at 0x7f6115e98c10>
when = 'setup'
reraise = (<class '_pytest.outcomes.Exit'>, <class 'KeyboardInterrupt'>)

    @classmethod
    def from_call(
        cls,
        func: "Callable[[], TResult]",
        when: "Literal['collect', 'setup', 'call', 'teardown']",
        reraise: Optional[
            Union[Type[BaseException], Tuple[Type[BaseException], ...]]
        ] = None,
    ) -> "CallInfo[TResult]":
        """Call func, wrapping the result in a CallInfo.
    
        :param func:
            The function to call. Called without arguments.
        :param when:
            The phase in which the function is called.
        :param reraise:
            Exception or exceptions that shall propagate if raised by the
            function, instead of being wrapped in the CallInfo.
        """
        excinfo = None
        start = timing.time()
        precise_start = timing.perf_counter()
        try:
>           result: Optional[TResult] = func()

/usr/lib/python3/site-packages/_pytest/runner.py:340: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/usr/lib/python3/site-packages/_pytest/runner.py:261: in <lambda>
    lambda: ihook(item=item, **kwds), when=when, reraise=reraise
/usr/lib/python3/site-packages/pluggy/_hooks.py:265: in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
/usr/lib/python3/site-packages/pluggy/_manager.py:80: in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
/usr/lib/python3/site-packages/_pytest/skipping.py:236: in pytest_runtest_setup
    skipped = evaluate_skip_marks(item)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

item = <Function test_datasheet_yamllint>

    def evaluate_skip_marks(item: Item) -> Optional[Skip]:
        """Evaluate skip and skipif marks on item, returning Skip if triggered."""
        for mark in item.iter_markers(name="skipif"):
            if "condition" not in mark.kwargs:
                conditions = mark.args
            else:
                conditions = (mark.kwargs["condition"],)
    
            # Unconditional.
            if not conditions:
                reason = mark.kwargs.get("reason", "")
                return Skip(reason)
    
            # If any of the conditions are true.
            for condition in conditions:
                result, reason = evaluate_condition(item, mark, condition)
                if result:
                    return Skip(reason)
    
        for mark in item.iter_markers(name="skip"):
            try:
                return Skip(*mark.args, **mark.kwargs)
            except TypeError as e:
>               raise TypeError(str(e) + " - maybe you meant pytest.mark.skipif?") from None
E               TypeError: Skip.__init__() got multiple values for argument 'reason' - maybe you meant pytest.mark.skipif?

/usr/lib/python3/site-packages/_pytest/skipping.py:190: TypeError
=========================== short test summary info ============================
ERROR tests/test_code_conventions.py::test_datasheet_yamllint - TypeError: Sk...
================== 637 passed, 258 skipped, 1 error in 43.69s ==================

```

--- a/tests/test_code_conventions.py
+++ b/tests/test_code_conventions.py
@@ -69,17 +69,6 @@ def test_free():
                 print("Suspicious `free` in {}:{}:{}".format(fn,no,line))
     assert okay, "'free' is used in some files.  These should be changed to 'OQS_MEM_secure_free' or 'OQS_MEM_insecure_free' as appropriate. If you are sure you want to use 'free' in a particular spot, add the comment '// IGNORE free-check' on the line where 'free' occurs."
 
-# Ensure that the datasheet .yml files are correctly formatted
-# TODO: Re-enable once yamllint is happy with the output produced by
-# scripts/format_docs_yaml.py (we could also specify a custom profile for
-# yamllint or use a different linter).
-@helpers.filtered_test
-@pytest.mark.skip()
-def test_datasheet_yamllint():
-    helpers.run_subprocess(
-        ['yamllint', 'docs/algorithms']
-    )
-
 if __name__ == "__main__":
     import sys
     pytest.main(sys.argv)

--- a/tests/test_code_conventions.py
+++ b/tests/test_code_conventions.py
@@ -74,7 +74,7 @@ def test_free():
 # scripts/format_docs_yaml.py (we could also specify a custom profile for
 # yamllint or use a different linter).
 @helpers.filtered_test
-@pytest.mark.skip(sys.platform.startswith("win"), reason="yamllint does not agree with current automatic formatting script.")
+@pytest.mark.skip()
 def test_datasheet_yamllint():
     helpers.run_subprocess(
         ['yamllint', 'docs/algorithms']

